java/src/main/java/ai/onnxruntime/OnnxRuntime.java

```diff
@@ -111,12 +121,12 @@ static synchronized void init() throws IOException {
    *
    * @param file The file to remove.
    */
-  private static void cleanUp(File file) {
+  private static void cleanUp(File file, boolean on_exit_only) {
```

Comment:
Java variable names are idiomatically camelCase not snake_case. For both this and the load method.

Reply:
Sorry, I didn't think about it. Fixed